Model/GetSourceSelectionResultFromOrder.php

```diff
@@ -82,7 +82,7 @@ public function execute(OrderInterface $order): SourceSelectionResultInterface
     /**
      * Get selection request items
      *
-     * @param OrderItemInterface[]|Traversable $orderItems
+     * @param iterable $orderItems
```

Comment:
Umm, so how come we're changing this to iterable? The docblock `OrderItemInterface[]` should imply this